lib/src/test/java/com/thenewboston/utils/Mocks.kt

```diff
@@ -365,6 +367,19 @@ object Mocks {
         80,
         "",
     )
+
+    fun postClean(clean: String = "cleaning") = Clean(
+        cleanLastCompleted = Some.dateTime,
+        cleanStatus = clean,
+        ipAddress = "20.188.56.203",
+        port = 80,
+        protocol = "http"
+    )
+
+    fun postCleanRequest() = PostCleanRequest(
+        data = Data(clean = "cleaning"),
```

Comment:
Should this not be "start" or "stop" as it says in the documentation?

Reply:
Done, thanks again